finance-portal/templates/deployment.yaml

```diff
@@ -52,6 +52,10 @@ spec:
             name: {{ template "finance-portal.fullname" . }}
         - secretRef:
             name: {{ template "finance-portal.fullname" . }}
+        volumeMounts:
+        - name: {{ template "finance-portal.fullname" . }}-secrets-volume
+          mountPath: /opt/finance-portal-backend-service/secrets
+          readOnly: true
```

Comment:
Wrap in a conditional block `{{- if .Values.JWSKeys }} .. content .. {{- end }}`